gulpfile.js/tasks/init.js

```diff
@@ -5,8 +5,8 @@ var projectPath = require('../lib/projectPath')
 var merge       = require('merge-stream')
 
 gulp.task('init', function() {
-  var rootStream = gulp.src('root/*')
-    .pipe(gulp.dest(projectPath()))
+  var rootStream = gulp.src(['root/.eslintrc.js', 'root/.gitignore', 'root/.nvmrc', 'root/.prettierignore', 'root/.stylelintrc'])
```

Comment:
i couldn't get globbing to work with `.` prefixed files, so had to list them out explicitly. if there is a better way, please let me know

Reply:
Did you try `'root/.*'`? `*` won't match files hidden with an initial `.`. (Doesn't look like it's needed in this case, but `'root/{.,}*'` should match every direct child of `root/`)